models/user.js

```diff
@@ -5,50 +5,44 @@ const Schema = mongoose.Schema;
 //mongoDB automatically generates an _id for every entry we can use this as the userID
 
 const user = new Schema({
-
   // email should have unique: true
 
-	username: {
-		type: String,
-		required: true,
-		index: true
-	},
-
-	age: {
-		type: String,
-		required: true
-	},
-
-	city: {
-		type: String,
-		required: true
-	},
-
-	email: {
-		type: String,
-		required: true
-	},
-
-	password: {
-		type: String,
-		required: true
-	},
-
-	registeredAt: {
-		type: Date,
-		required: true,
-		default: new Date()
-	},
-
-	lastLoggedIn: {
-		type: Date
-	},
-
-	picture: {
-		type: String,
-		required: true
-	}
-
+  username: {
+    type: String,
+    required: true,
+    index: true,
+  },
+
+  age: {
+    type: String,
+  },
+
+  city: {
+    type: String,
+  },
+
+  email: {
+    type: String,
```

Comment:
i think we should set here unique: true and also with the username unique: true, i still see my comment on top 
should we just allow unique usernames ? or unique usernames and emails ?